.github/workflows/pullrequests.yml

```diff
@@ -113,3 +113,46 @@ jobs:
 
       - name: Report Code Coverage
         uses: codecov/codecov-action@v3
+  e2e-test:
+    if: ${{ github.repository_owner == 'cloudflare' }}
```

Comment:
How would you feel about only running these on PRs that change wrangler code?

Reply:
I don't really want to run these on PRs at all, given how long they'll take. I think they'll only run on the `main` branch (they're only enabled for the pull request workflow right now so that this PR is vaguely testable)